templates/project/app/build.gradle

```diff
@@ -248,8 +252,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
```

Comment:
I was under the impression we were NOT doing this change 🤔 

Also, can you add information to the commit message referencing the commits/ PRs we are getting this from (I believe this is an opened PR on cordova-android, if IIRC what you showed me)?

Reply:
Please ignore this. Slipped somehow. Amateur hour.